XmlFormatter/src/Windows/HotfolderEditor.cs

```diff
@@ -0,0 +1,227 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Windows.Forms;
+using XmlFormatter.src.Enums;
+using XmlFormatter.src.Hotfolder;
+using XmlFormatter.src.Interfaces.Formatter;
+using XmlFormatter.src.Interfaces.Hotfolder;
+
+namespace XmlFormatter.src.Windows
+{
+    /// <summary>
+    /// This Window is used to create or edit a hotfolder configuration
+    /// </summary>
+    public partial class HotfolderEditor : Form
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5053278)